extensions/amp-3q-player/0.1/validator-amp-3q-player.protoascii

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-3q-player
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-3q-player extension .js script"
+  satisfies: "amp-3q-player extension .js script"
+  mandatory_parent: "HEAD"
+  requires: "amp-3q-player"
```

Comment:
Instead of `requires` here lets use this instead: `extension_unused_unless_tag_present: "amp-3q-player"`. This also means we get to remove the `satisfies: "amp-3d-player"` below.